pkg/installation/installation.go

```diff
@@ -594,7 +594,7 @@ func (i *Installation) waitForInstaller() error {
 		case <-timeout:
 			i.currentStep.Failure()
 			if err := i.printInstallationErrorLog(); err != nil {
-				fmt.Println("Error fetching installation error log, please manually check the status of the cluster.")
+				fmt.Println("Error fetching installation error log: ", err, "\nPlease manually check the status of the cluster")
```

Comment:
Please use `fmt.Printf` here if you want to print a string containing the error.
This way is not clear what will be printed.